### PR TITLE
3392 filter error between inventory lists

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
Filters of `is not None` break on the new inventory list.

#### What's this PR do?
Fixes filtering for `is not None` for default canonical columns

#### How should this be manually tested?
Go to inventory legacy and search filter a column that has empty values using `!=""` (double quotes, not single)
Navigate to inventory list and confirm there are no errors

#### What are the relevant tickets?
#3392

#### Screenshots (if appropriate)
Screen Shot 2022-07-25 at 2 58 32 PM

### CODE REVIEW

**Comment by perryr16** [comment]
Please leave this ticket open as it does not address the error when filtering on extra data columns. Because extra data columns values are within a nested dictionary, checking for `is not None` will be more challenging

File: seed/search.py [hunk 1]
```diff
@@ -548,6 +549,9 @@ def build_view_filters_and_sorts(filters: QueryDict, columns: list[dict]) -> tup
     new_filters = Q()
     annotations = {}
     for filter_expression, filter_value in filters.items():
+        if filter_expression[-4:] == '__ne' and filter_value == '':
+            filter_expression = filter_expression.replace('__ne', '__isnull')
+            filter_value = False
```
Comment (perryr16) [inline seed/search.py:554]: I don't believe this is the most elegant solution, but it should work until that solution is found

Comment (Ryoken) [reply seed/search.py:554]: Eew, yeah, that's pretty gross.  But based on what we talked about I can't see a better way to do this.

**Review by Ryoken — APPROVED** [review]

**Comment by perryr16** [comment]
Just to document another option. Currently, we are always using the `filter` function, we could add an argument and a conditional to `exclude` on specific queries

**Review by nllong — APPROVED** [review]
nice, fix works. 🥳

**Pull request merged**